compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -886,7 +886,7 @@ class Allocator {
 
   void setTrackAllocationStacks(bool Track) {
     initThreadMaybe();
-    if (getFlags()->allocation_ring_buffer_size == 0) {
+    if (getFlags()->allocation_ring_buffer_size < 1) {
```

Comment:
Can we do this with `<= 0`? I think the intention will be more clear

In addition, if we can make `allocation_ring_buffer_size` unsigned, `== 0` will be better

Reply:
> Can we do this with <= 0? I think the intention will be more clear

OK

> In addition, if we can make allocation_ring_buffer_size unsigned, == 0 will be better

I would rather not change the parser for this. There is precedence to making negative values disable, for `release_to_os_interval_ms`.